partner_education_views/models/education.py

```diff
@@ -1,14 +1,27 @@
-from odoo import models, fields, api
+from odoo import models, fields, api, _
 
 
 class ResPartnerEducation(models.Model):
     _name = "res.partner.education"
 
+    partner_id = fields.Many2one(comodel_name="res.partner")
     sun_id = fields.Many2one(comodel_name='res.sun',
                                string='SUN Code')
     education_level_id = fields.Many2one(
         comodel_name="res.partner.education.education_level",
         string="Education level")
     foreign_education = fields.Boolean(string="Foreign education")
     foreign_education_approved = fields.Boolean(
-        string="Foreign education approved")
\ No newline at end of file
+        string="Foreign education approved")
+
+    @api.multi
+    def name_get(self):
+        res = super(ResPartnerEducation, self).name_get()
+        data = []
+        for education in self:
```

Comment:
String concatenation is expensive a better method would be to construct a list and then join it.
Various ways to do this:
 "".join(my_list)
"".join(["string1" or "", string2 or ""])
f'{education.sun_id.name or ""},{education.education_level_id.name or _("None")}'